drivers/integration/linux/linux.go

```diff
@@ -459,8 +459,10 @@ func (d *driver) Create(
 	volumeType := d.volumeType()
 	optsNew.Type = &volumeType
 	io, _ := strconv.Atoi(d.iops())
```

Comment:
I know it's not part of this PR, but I hate seeing this err suppressed. It doesn't appear that the syntax for this config value has been validated previously, so it's possible this conversion should fail. Something worth addressing?